lib/core/utils/is-html-element.js

```diff
@@ -1,137 +1,20 @@
-const htmlTags = [
-  'a',
-  'abbr',
-  'address',
-  'area',
-  'article',
-  'aside',
-  'audio',
-  'b',
-  'base',
-  'bdi',
-  'bdo',
-  'blockquote',
-  'body',
-  'br',
-  'button',
-  'canvas',
-  'caption',
-  'cite',
-  'code',
-  'col',
-  'colgroup',
-  'data',
-  'datalist',
-  'dd',
-  'del',
-  'details',
-  'dfn',
-  'dialog',
-  'div',
-  'dl',
-  'dt',
-  'em',
-  'embed',
-  'fieldset',
-  'figcaption',
-  'figure',
-  'footer',
-  'form',
-  'h1',
-  'h2',
-  'h3',
-  'h4',
-  'h5',
-  'h6',
-  'head',
-  'header',
-  'hgroup',
-  'hr',
-  'html',
-  'i',
-  'iframe',
-  'img',
-  'input',
-  'ins',
-  'kbd',
-  'keygen',
-  'label',
-  'legend',
-  'li',
-  'link',
-  'main',
-  'map',
-  'mark',
-  'math',
-  'menu',
-  'menuitem',
-  'meta',
-  'meter',
-  'nav',
-  'noscript',
-  'object',
-  'ol',
-  'optgroup',
-  'option',
-  'output',
-  'p',
-  'param',
-  'picture',
-  'pre',
-  'progress',
-  'q',
-  'rb',
-  'rp',
-  'rt',
-  'rtc',
-  'ruby',
-  's',
-  'samp',
-  'script',
-  'section',
-  'select',
-  'slot',
-  'small',
-  'source',
-  'span',
-  'strong',
-  'style',
-  'sub',
-  'summary',
-  'sup',
-  'svg',
-  'table',
-  'tbody',
-  'td',
-  'template',
-  'textarea',
-  'tfoot',
-  'th',
-  'thead',
-  'time',
-  'title',
-  'tr',
-  'track',
-  'u',
-  'ul',
-  'var',
-  'video',
-  'wbr'
-];
+import standards from '../../standards';
 
 /**
  * Verifies that if a given html tag is valid
  * @method isHtmlElement
  * @memberof axe.utils
- * @param htmlTag htmlTag to check if valid
+ * @param {HTMLElement|VirtualNode} node node to check if valid
  * @return {Boolean} true/ false
  */
 function isHtmlElement(node) {
+  const nodeName = node.props?.nodeName ?? node.nodeName.toLowerCase();
+
   if (node.namespaceURI === 'http://www.w3.org/2000/svg') {
     return false;
   }
 
-  return htmlTags.includes(node.nodeName.toLowerCase());
+  return Object.keys(standards.htmlElms).includes(nodeName);
```

Comment:
Don't think you need to create an array here:

```suggestion
  return !!standards.htmlElms[nodeName];
```

The other thing about this, if someone for some reason wanted to unset an HTML element with `axe.configure({ standards })`, they they wouldn't be able to, since this still returns true, even if the prop on `htmlElms` is undefined or null.